pkg/datapath/linux/utime/cell.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package utime
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cilium/cilium/pkg/controller"
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/maps/configmap"
+)
+
+const (
+	syncControllerName     = "sync-utime"
+	syncControllerInterval = 1 * time.Minute
+)
+
+// Cell initializes and manages the utime offset synchronization.
+var Cell = cell.Module(
+	"utime",
+	"Synchronizes utime offset between userspace and datapath",
+
+	cell.Invoke(initUtimeSync),
+)
+
+func initUtimeSync(lifecycle hive.Lifecycle, configMap configmap.Map) {
+	controllerManager := controller.NewManager()
```

Comment:
@joamaki would it be worth to have a provider for the controller manager?

Reply:
What do you think of https://github.com/cilium/cilium/pull/24558?

Reply:
interesting - will take a look.

i think something like this could help in this case - and could be refactored in the future (once its merged)